### PR TITLE
Feature(wren-ai-service): Ask pipeline load test

### PR DESCRIPTION
**Pull request by imAsterSun**
### Locust User Config ###
- Adjust number of users in `config_users.json` 
- The total number of users should equal to the `users` config in `locust.conf`

### Log File ###
- Review test log in `outputs/locust_report_{timestamp}.log`

### CODE REVIEW

File: wren-ai-service/tests/config_users.json
Comment (paopa) [inline wren-ai-service/tests/config_users.json:1]: I'm not familiar with Locust. what is the work for this json file? thanks!

File: wren-ai-service/tests/locust.conf
Comment (paopa) [inline wren-ai-service/tests/locust.conf:1]: it's just a nice-to-have comment. I saw the document of locust that supports placing the config in pyproject.toml. we might consider using this. reference: https://docs.locust.io/en/stable/configuration.html#pyproject-toml

**Review by paopa — COMMENTED** [review]
How about placing all Locust files into a `tools` folder? The structure would look like this:

```
wren-ai-service/
├── src/
├── test/
└── tools/
```

I think we could avoid to place the packages in the src, which are not used in runtime.

File: wren-ai-service/tests/config_users.json
Comment (cyyeh) [reply wren-ai-service/tests/config_users.json:1]: or how about this
wren-ai-service/
- src/
- tests/
  - pytest/
  - locust/

**Comment by cyyeh** [comment]
@imAsterSun thanks for your hard work, after the review and modification if needed is finished, I suppose there is one pipeline left for locust implementation: AskDetails, right?

File: wren-ai-service/README.md [hunk 1]
```diff
@@ -42,9 +42,19 @@ Please read the [documentation](https://docs.getwren.ai/concept/wren_ai_service)
   - restart wren ai service
   - check `outputs` folder for the evaluation result file
 - to run the load test
-  - in the demo folder, run `make prepare` to start the docker containers
-  - in the demo folder, run `make ai` to start the ai service
+  - Choose between the 2 setup options:
+      - Setup option 1: demo env setup
+        - in the demo folder, run `make prepare` to start the docker containers
+        - in the demo folder, run `make ai` to start the ai service
```
Comment (cyyeh) [inline wren-ai-service/README.md:48]: I suggest we only provide the second setup, since we don't provide the way to ingest dataset for the first setup.

Comment (imAsterSun) [reply wren-ai-service/README.md:48]: Sure, let me adjust it.

File: wren-ai-service/tests/config_users.json
Comment (imAsterSun) [reply wren-ai-service/tests/config_users.json:1]: > I'm not familiar with Locust. what is the work for this json file? thanks!

For setting the number of each user, we can use this to adjust test spec

**Review by paopa — APPROVED** [review]
LGTM

File: wren-ai-service/README.md [hunk 2]
```diff
@@ -42,9 +42,17 @@ Please read the [documentation](https://docs.getwren.ai/concept/wren_ai_service)
...
-  - in the demo folder, run `make prepare` to start the docker containers
-  - in the demo folder, run `make ai` to start the ai service
+  - setup `DATASET_NAME` in `.env.dev`
+  - adjust test config if needed
+    - adjust test config in pyproject.toml `tool.locust` section
+    - adjust user count in `tests/locust/config_users.json`
+  - in wren-ai-service folder, run `make run-all` to start the docker containers
+  - in wren-ai-service folder, run `make start` to start the ai service
   - run `make load-test`
+  - check reports in /outputs folder, there are 3 files with filename **locust_report_{test_timestamp}**:
```
Comment (cyyeh) [inline wren-ai-service/README.md:52]: check reports in /outputs/locust folder

**Review by cyyeh — COMMENTED** [review]
only minor comment

File: wren-ai-service/README.md (hunk 2 above)
Comment (imAsterSun) [reply wren-ai-service/README.md]: Thanks! README updated.

**Review by paopa — APPROVED** [review]

**Pull request merged**